public/icons/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Alert } from "./alert";
+export { ArrowRight } from "./arrowRight";
+export { GasOutline } from "./gasOutline";
```

Comment:
per il naming dei file quale convention usiamo in questa repo?

https://www.notion.so/belkadigital/Components-API-Style-Guide-1b63d3c013b74a419af38ba21047f3be?pvs=4#16eed8e0262649378bf9e72379e1bf04

Reply:
Temo di aver fatto un mix. Devo rinominare i file dei componenti con la maiuscola, anche i file di Storybook (es: `Card.stories.tsx`), torna? Il resto rimane lowercase (types, styles ecc)

Reply:
> File Naming: use kebab-case for naming files and directories to maintain consistency.

nella doc su notion leggo che file e cartelle sono in kebab case, quindi in questo caso file di storybook sono già a posto, andrebbero aggiornate solo le icone

```
gas-outilne.tsx
arrow-right.tsx
```

però taggo anche @mattiafilardibelka per sicurezza

Reply:
Anche a me torna quello che ha detto @anas-araid-belka 👍